src/usr/local/www/diag_reboot.php

```diff
@@ -102,28 +117,40 @@ function startCountdown() {
 <?php
 else:
 
-?>
+$form = new Form(false);
+
+$help = 'Click "Normal reboot" to reboot the system immediately, "Reboot with Filessystem Check" to reboot and run filesystem check';
+$modeslist = ['Reboot' => 'Normal reboot', 'FSCKReboot' => 'Reboot with Filesystem Check'];
+if (!is_module_loaded("zfs.ko")) {
+	$help .= ' or "Reroot" to stop processes, remount disks and re-run startup sequence';
+	$modeslist += ['Reroot' => 'Reroot'];
+	} 
+$help .= '.';
+
+$section = new Form_Section('Select reboot method');
+
+foreach (auth_get_authserver_list() as $key => $auth_server) {
```

Comment:
Why is this here? It doesn't appear to be used for anything.

Reply:
removed